client/src/components/byline-autocomplete/index.jsx

```diff
@@ -1,13 +1,14 @@
 // External dependencies.
-import React from 'react';
 import { useState, useEffect } from '@wordpress/element';
+import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import Autocomplete from 'react-autocomplete';
 
 // Hooks.
 import { useDebounce } from '@uidotdev/usehooks';
 
 const BylineAutocomplete = ({
+  id,
```

Comment:
I think these components could benefit from a `forwardRef`. But I don't have a use case for it yet. 🤔